docker-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Migrate database
+python manage.py migrate
+
+exec $@
```

Comment:
you are correct. It allows for other optional commands and configs to be passed into the execute. For some reason it did not work when I removed that line. I suspect docker requires you to "leave that door open"